tests/integration/test_clip.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+from legate.core import LEGATE_MAX_DIM
+from utils.generators import mk_seq_array
+
+import cunumeric as num
+
+# cunumeric.clip(array, a_min, a_max, out=None, **kwargs)
+
+
+@pytest.mark.xfail
+def test_none_array():
+    expected_exc = TypeError
+    with pytest.raises(expected_exc):
+        np.clip(None, a_min=0, a_max=0)
+    with pytest.raises(expected_exc):
+        # cunumeric raises
+        # AttributeError: 'NoneType' object has no attribute 'clip'
+        num.clip(None, a_min=0, a_max=0)
+
+
+def test_empty_array():
+    res_np = np.clip([0], a_min=0, a_max=0)
+    res_num = num.clip([0], a_min=0, a_max=0)
+    assert np.array_equal(res_np, res_num)
```

Comment:
This test seems mis-named? `[0]` is not an empty array

Reply:
yes, the array should be [ ]